### DISCUSSION TITLE
Port data and backup from Windows to Linux

### DISCUSSION THREAD

**Discussion by alexfan8956** [nextcloud/all-in-one#4356] [Questions] [answered]
Hi everyone,

I have got a temporary NC AIO setup inside the docker desktop on my windows machine while I am waiting to get some free time to setup my linux server. My questions here are:
- any official guide to port windows setup to linux? if not, what are the steps? I see that I might need to port the volumes, data dir and backup fir
- will the data and backup still work if I copy over to linux or the content of those files are windows specific?

**Comment by szaimen** [comment, marked as answer]
Yes, a backup taken on docker for windows with aios included borg-backup solution can be restored on a fresh docker for linux aio installation.

**Reply by alexfan8956** [reply]
Thanks a lot for the reply. When I need to migrate the data to new linux server, should I just use the backup or use the mapped data dir which contains the most up to date content?

**Reply by szaimen** [reply]
Best to use the backup to restore it
